hal/halx86/acpi/halacpi.c

```diff
@@ -994,8 +994,8 @@ HalpBuildAcpiResourceList(IN PIO_RESOURCE_REQUIREMENTS_LIST ResourceList)
 
         /* Get the interrupt number */
         Interrupt = HalpPicVectorRedirect[HalpFixedAcpiDescTable.sci_int_vector];
-        ResourceList->List[0].Descriptors[0].u.Interrupt.MinimumVector = Interrupt;
-        ResourceList->List[0].Descriptors[0].u.Interrupt.MaximumVector = Interrupt;
+        ResourceList->List[0].Descriptors[0].u.Interrupt.MinimumVector = Interrupt + PRIMARY_VECTOR_BASE;
```

Comment:
PRIMARY_VECTOR_BASE is the processor's view of the IRQ vector (int 0x30), whereas an IO_RESOURCE_REQUIREMENTS list always refers to raw resources (IRQ 0). So this translation doesn't belong here. Since ROS doesn't have translators & arbiters, I'm not exactly sure where it does belong though :\

Reply:
Are there any plans to implement resource translators/arbiters in the near future?

Reply:
Not to my knowledge. Regardless, 'here' is not the answer to where it belongs.